content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
```

Comment:
What does `Category: pages` do exactly? I only added because I saw other pages (`mua.md`, `office-policy`.md etc) had them as well. (I'll take them out now)